chunky/src/java/se/llbit/chunky/renderer/Postprocess.java

```diff
@@ -36,6 +36,12 @@
     @Override public String toString() {
       return "Tonemap operator 1";
     }
+  },
+  TONEMAP2 {
+    @Override public String toString() { return "ACES filmic tone mapping"; }
```

Comment:
I prefer to not use this single-line style of method declaration. It's a personal preference, but it's better to keep everything consistent. See for example the other `toString()` methods in this file.